src/agent/configuration_parser/include/configuration_parser.hpp

```diff
@@ -24,6 +24,9 @@ namespace configuration
         /// @brief Holds the parsed YAML configuration.
         YAML::Node m_config;
 
+        /// @brief Holds the location of the configuration file.
+        std::filesystem::path m_config_file_path;
```

Comment:
Name convention should be `m_configFilePath`.

Reply:
Done!